packages/datagateway-common/package.json

```diff
@@ -10,11 +10,17 @@
     "@date-io/date-fns": "^1.3.11",
     "@material-ui/pickers": "^3.2.6",
     "@types/lodash.debounce": "^4.0.6",
+    "axios": "^0.19.2",
```

Comment:
can the new packages in this file match the versions in the other plugins? Since `common` is imported by the others, I don't want it requiring a newer version for no reason and increasing the bundle size.

(based off of `table` dependencies since they're often the oldest)
`axios` -> `^0.19.0`
`connected-react-router` -> `^6.5.2`
`loglevel` -> `^1.6.3`
`react-redux` -> `^7.1.3`
`redux` -> `^4.0.4`